### PR TITLE
update compiler to v1.1.102, exclude `$referrerpolicy` rules for unsupported platforms

### PR DESCRIPTION
**Pull request by slavaleleka**
1 run `yarn install --force` to make sure that latest compiler version is installed
2 build filters — `$referrerpolicy` rules should not be invalid anymore

https://github.com/AdguardTeam/FiltersCompiler/issues/191

### CODE REVIEW

File: custom_platforms.js [hunk 1]
```diff
@@ -76,6 +76,8 @@ module.exports = {
                 ",hls=",
                 "\\$jsonprune=",
                 ",jsonprune=",
+                "\\$referrerpolicy=",
+                ",referrerpolicy=",
```
Comment (Alex-302) [inline custom_platforms.js:80]: https://adguard.com/kb/ru/general/ad-filtering/create-own-filters/#referrerpolicy-modifier:~:text=com/abcd.html%5E-,%24referrerpolicy,-%D0%BE%D1%82%D0%BA%D0%BB%D1%8E%D1%87%D0%B0%D0%B5%D1%82%20%D0%B2%D1%81%D0%B5%20%D0%BF%D1%80%D0%B0%D0%B2%D0%B8%D0%BB%D0%B0

Will not be excluded
`@@||example.com/abcd.html^$referrerpolicy`

```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

File: custom_platforms.js [hunk 2]
```diff
@@ -107,6 +109,8 @@ module.exports = {
...
```
Comment (Alex-302) [inline custom_platforms.js:113]:
```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

File: custom_platforms.js [hunk 3]
```diff
@@ -139,6 +143,8 @@ module.exports = {
...
```
Comment (Alex-302) [inline custom_platforms.js:147]:
```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

File: custom_platforms.js [hunk 4]
```diff
@@ -211,6 +217,8 @@ module.exports = {
                 "\\$webrtc",
                 "\\$csp",
                 "\\$network",
+                "\\$referrerpolicy=",
+                ",referrerpolicy=",
```
Comment (Alex-302) [inline custom_platforms.js:221]:
```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

File: custom_platforms.js [hunk 5]
```diff
@@ -261,6 +269,8 @@ module.exports = {
...
```
Comment (Alex-302) [inline custom_platforms.js:273]:
```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

File: custom_platforms.js [hunk 6]
```diff
@@ -331,6 +341,8 @@ module.exports = {
                 ",redirect=",
                 "\\$redirect-rule=",
                 ",redirect-rule=",
+                "\\$referrerpolicy=",
+                ",referrerpolicy=",
```
Comment (Alex-302) [inline custom_platforms.js:345]:
```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

File: custom_platforms.js [hunk 7]
```diff
@@ -374,6 +386,8 @@ module.exports = {
                 "#\\$#@media ",
                 "\\$hls=",
                 ",hls=",
+                "\\$referrerpolicy=",
+                ",referrerpolicy=",
```
Comment (Alex-302) [inline custom_platforms.js:390]:
```suggestion
                "\\$referrerpolicy",
                ",referrerpolicy",
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**